spec/tree_spec.rb

```diff
@@ -1,14 +1,48 @@
 require 'rspec'
 require 'tree'
 
-describe 'Tree' do
+describe Tree do
   it 'should be a Class' do
-    expect(described_class.is_a? 'Class').to be_true
+    expect(described_class.is_a? Class).to eq true
+  end
+
+  it 'should have height' do
+    tree = Tree.new
+    expect(tree.height).to_not eq nil
+  end
+
+  it 'should age' do
+    tree = Tree.new
+    tree.age!
+    expect(tree.height).to eq 1
+    expect(tree.age).to eq 1
+    expect(tree.apples).to_not eq []
+  end
+
+  it 'should add apples' do
+    tree = Tree.new
+    tree.add_apples
+    expect(tree.apples).to_not eq []
+  end
```

Comment:
For a lot of these tests, you might do some thinking about what the subject of the test really is.  Here, for instance, the subject is really the `add_apples` method, rather than the `Tree` class itself.  So using a nested describe block would be a good way to represent that.

Also, think about whether this is the right test.  Sometimes you might want to call `add_apples` on a tree that already had apples, in which case the fact that `tree.apples` is not empty is pretty meaningless.